backend/src/routes/router.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express')
+const main_router = express.Router()
+
```

Comment:
if you want you can do `const products = require('../../db.json')` and on line 17 in the 'getAllInventort' you change msg to 'products'

Reply:
Hey, sorry about the lack of progress I have been working on creating a fully functioning backend server for categories and products on a separate private project, I haven't had a chance to update this backend yet for phase III of our lab. Thanks for the tip that would definately work!!